README.md

```diff
@@ -651,6 +652,31 @@ As such, HTTP/1.1 response messages will automatically include a
 `Connection: close` header, irrespective of what header values are
 passed explicitly.
 
+### Middleware
+
+Middleware can be added to the server using [`MiddlewareRunner`](src/MiddlewareRunner.php)
+instead of the `callable`. A middleware is expected to adjure the following rules:
```

Comment:
@jsor suggested `adhere` instead?

Reply:
@clue whoops thought I fixed that one, fixed it, squashed it, and pushed it again :tada: